tilavarauspalvelu/models/reservation_unit/queryset.py

```diff
@@ -32,16 +31,6 @@
 
 
 class ReservationUnitQuerySet(SearchResultsQuerySet):
-    def scheduled_for_publishing(self) -> Self:
```

Comment:
When removing unused code, its good to do it in a separate commit or mention that in the commit message 🤓